test-avocado/compiletest.sh

```diff
@@ -16,7 +19,7 @@ test -d $PACKAGE/$BUILD && /bin/rm -r $PACKAGE/$BUILD
 cd $PACKAGE
 mkdir -p $BUILD
 cd $BUILD
-../autogen.sh --prefix=/usr --enable-maintainer-mode --enable-debug
+../autogen.sh --prefix=/usr --libdir=/usr/lib$ARCH64 --enable-maintainer-mode --enable-debug
```

Comment:
I'm fine with that, if you're going to maintain the alternate build system. Significant investment needs to be made to keep it in sync with make-rpms however. In particular:

 * Skipping installation of debug files (See %install target in cockpit.spec)
 * SELinux policy installation (when necessary)
 * PAM file installation (I think you got this already actually)
 * ... and I'm sure there's more.

As far as having 'make-deb' and so on, once we do have CI running against those operating systems (and someone invested in keeping on top of it), then it makes sense to have those tools and to add the complexity of doing non-RPM systems in our CI, until then it's a lot of unnecessary overhead.